INSTALLATION.md

```diff
@@ -0,0 +1,156 @@
+# Walkthrough 
+
+This is a walkthrough for the OCP worker on VMs project, that can be found here: [ocp-lab-setu]() https://github.com/oVirt/ocp-lab-setup)
+
+## Setting up the environment
+
+### Get an inventory
+```
+./05-get-inventory.sh http://quads.rdu2.scalelab.redhat.com/cloud/cloud09_ocpinventory.json
+```
+We removed the first 5 entries, since we need those hosts as masters and infra nodes which resulted in the `hosts` file.
+```
+e21-h20-b03-fc640.rdu2.scalelab.redhat.com,eno1,ens2f0,ens2f1
+e21-h20-b04-fc640.rdu2.scalelab.redhat.com,eno1,ens2f0,ens2f1
+```
+We need to distribute our public key so we can work on the nodes without the need to authenticate every time. Make sure it uses the correct key in case you have more than just one in the user's .ssh directory.
+
+```
+./10-ssh-keys.ssh
+```
+This copies your ssh key ~/.ssh/id_rsa.pub key to the nodes in the `hosts` file.
+
+Now we need to install the required packages on all the nodes, the engine as well as the VM hosts:
+```
+./15-install-packages.sh http://bob.eng.lab.tlv.redhat.com/builds/4.5/rhv-4.5.3-1/api/ 
+[LAB] Install engine
+...
+[LAB] Install hosts
+...
+[LAB] oVirt packages done
+```
+
+No longer needed:
+~~**Make sure to reboot the nodes after this step!**~~
+
+```bash
+for i in $(cat engine) $(cat hosts); do ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -o User=root $i reboot; done
+```
+
+
+The next step takes care of network configuration, dnsmasq for RHV and the SD data share. If you want to set a cluster name different to the default of *perfscale*, please do so in the **common_funcs** file. Also make sure the domain is set correctly. This information relates to the name and domain you set in your OCP cluster.
```

Comment:
oh, right, it likely makes sense to mention this at the very beginning. Also the number of masters/workers.